pages/applications/index.js

```diff
@@ -31,12 +31,12 @@ const Applications = () => {
       ]);
     }
 
-    if (user && user.roles.includes('community-curator')){
+    if (user){
       loadData();
     }
   }, [user, platform]);
 
-  if (!user || (!user.roles.includes('community-curator') && !user.roles.includes('admin'))) {
```

Comment:
This shouldn't be removed.

Reply:
My bad I was trying to see the content I forgot to put it back

Reply:
can we get this updated please?